assets/src/application/ui/input/Button/style.scss

```diff
@@ -162,6 +162,20 @@
 		}
 	}
 
+	/* ACTIVE */
+
+	&.ee-btn--is-active {
```

Comment:
added `is-active` to discriminate between the `active` state of the button, might consider a better naming here